inject.js

```diff
@@ -871,10 +900,20 @@ function injectScriptForSite() {
 
 function seek(mediaTag, seekSeconds) {
   switch (true) {
-    case location.hostname == "www.netflix.com":
-      window.postMessage({action: "videospeed-seek", seekMs: seekSeconds * 1000}, "https://www.netflix.com");
+    case location.hostname === "www.netflix.com":
+      window.postMessage(
+        { action: "videospeed-seek", seekMs: seekSeconds * 1000 },
+        "https://www.netflix.com"
+      );
+      break;
+    case location.hostname === "drive.google.com":
+      window.postMessage(
+        { action: "videospeed-seek", seekMs: seekSeconds * 1000 },
+        "https://drive.google.com"
+      );
       break;
     default:
+      // Directly update the media tag if it's neither Netflix nor Google Drive
       mediaTag.currentTime += seekSeconds;
```

Comment:
Hmm, isn't the default branch doing exactly the same thing?

Reply:
the extension is not working for the videos stored in google drive... so i thought this was the reason...

Reply:
Hmm, it's literally doing the same operation:

```
videoElement.currentTime += event.data.seekMs / 1000;
```

Something else is amiss here.